plugins/modules/swim_intent.py

```diff
@@ -169,11 +193,24 @@
         type: dict
         suboptions:
           device_role:
-            description: Device Role. Permissible Values ALL, UNKNOWN, ACCESS, BORDER ROUTER,
+            description: Device Role and  permissible Values are ALL, UNKNOWN, ACCESS, BORDER ROUTER,
```

Comment:
Defines the device role, with permissible values including ALL, UNKNOWN, ACCESS, BORDER ROUTER, DISTRIBUTION, and CORE.

Reply:
added.